config/initializers/active_admin.rb

```diff
@@ -201,7 +201,7 @@
   #
   # config.create_another = true
 
-  # == Register Stylesheets & Javascripts
```

Comment:
I would revert the ones in this file so as not to cause unnecessary conflicts when upgrading versions.

Reply:
Good idea! Done.